src/github.ts

```diff
@@ -50,4 +52,35 @@ function EnhancedGitHubClient (options) {
   return octokit
 }
 
-module.exports = EnhancedGitHubClient
+export interface Options extends Octokit.Options {
+  debug: boolean
+  logger: Logger
+  limiter?: any
+}
+
+export interface OctokitRequestOptions {
+  method: string
+  url: string
+  headers: any
+}
+
+export interface OctokitResult {
+  meta: {
+    status: string
+  }
+}
+
+export interface OctokitError {
+  code: number
+  status: string
+}
+
+export interface OctokitWithPagination extends Octokit {
```

Comment:
Could this add `paginate` to the existing `Octokit` interface instead of declaring a new interface?